exercises/knapsack/description.md

```diff
@@ -0,0 +1,10 @@
+Given a set of items, each with a weight, w and a value, v, determine the number of each item to include in a collection so that the total weight is less than or equal to a given limit and the total value is as large as possible
+
+Knapsack Max weight: W = 10 
+Total items: N = 4
```

Comment:
Total items (N) doesn't seem necessary to specify as an input; it can be inferred from the sizes of `v[]` and `w[]`.

If I think about how this problem would be implemented in a variety of languages:

Some languages will benefit from explicitly specifying N.
Some will not.
I predict that languages that benefit from explicitly specifying N are in the minority. This says how much benefit we get.

If this repository specifies N in canonical-data.json, the reviewers of this file will then be responsible for making sure it matches the lengths of `v[]` and `w[]`, with the associated costs.

An automated reviewer of course would be more able to bear this cost. Note that https://github.com/exercism/problem-specifications/issues/460 indicates a lack of interest in setting up an automated reviewer.

Reply:
Yup understood. Modified this as well.